draft-ietf-jsonpath-base.md

```diff
@@ -219,10 +219,11 @@ Location:
 
 Node:
 : The pair of a value along with its location within the argument.
-  Member names do not have locations and so do not have nodes.
-  Members are not values and so do not have nodes.
   Note that a node is either the root node (defined below) or one of
   its descendants (also defined below).
+  Member names and members cannot be selected by a query, but a member
```

Comment:
The only weird part here for me is that members aren't even values, so you cannot build the pair the node is supposed to be.

Reply:
Yes, in fact it is a current drawback of JSONPath, that we cannot select values by their member names (multiply addressed in the past).
```
{""foo":"bar"}

$[?nameOf(@) == 'foo']
```
Extension points for rescue ...

Reply:
Perhaps we need to more frequently remind the distinction:

member ➔ name/value pair
node  ➔ value along with its location in the argument

Reply:
> Perhaps we need to more frequently remind the distinction:
> 
> member ➔ name/value pair
> node ➔ value along with its location in the argument

I've included a reminder of what a member is in the new section introduced by 8da0011da49abb742ff22515c89560901e07b2ac.

We should check the rest of the spec to see if there are usages of "member" and "node" for which readers may need reminding of the definitions.